airflow-ctl/src/airflow/ctl/__main__.py

```diff
@@ -23,7 +23,7 @@
 
 import argcomplete
 
-from airflowctl.ctl import cli_parser
+from airflow.ctl.cli import cli_parser
```

Comment:
Just dropping a "request changes" review here so we don't forget to move this back to `airflowctl`.

Reply:
Reverted back to `airflow-ctl/src/airflowctl` :)